plugins/CoreHome/angularjs/common/services/piwik-api.js

```diff
@@ -49,7 +49,8 @@ var hasBlockedContent = false;
         function withTokenInUrl()
         {
             postParams['token_auth'] = piwik.token_auth;
-            postParams['force_api_session'] = '1';
+            // When viewing a widgetized report there won't be any session that can be used, so don't force session usage
+            postParams['force_api_session'] = (piwik.broadcast.getValueFromUrl('module') !== 'Widgetize') ? 1 : 0;
```

Comment:
@sgiehl wouldn't we need to look if `force_api_session` is present in the original URL or not or something? Like it would mean you can't use the widgetized api with the session API token. Not sure if we do use this somewhere or not in the UI or somewhere?

Reply:
Guess makes sense to check that. will change that

Reply:
@tsteur updated the code so it respects if force_api_session was already set. I'm not able to test the changes right now. Feel free to do that if needed otherwise I'll do that tomorrow